test/parallel/test-stream-preprocess.js

```diff
@@ -0,0 +1,71 @@
+﻿/******************************************************************************/
+'use strict';
+/******************************************************************************/
+const common = require('../common');
+const assert = require('assert');
+
+const fs = require('fs');
+const path = require('path');
+const rl = require('readline');
+/******************************************************************************/
```

Comment:
This is a minor nit, but in keeping with the style of most of the other tests, it's probably best to just replace this line with a blank line and remove the other two earlier asterisk lines completely.